src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -1,7 +1,5 @@
-from panoptes.utils import error
```

Comment:
This is being removed because `error` is not being used?

Reply:
Yeah, it's not being used